.github/workflows/flatpak-build.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         with:
           node-version: '18'
       - name: Install modules.
-        run: npm i --legacy-peer-deps
+        run: npm install --global yarn && yarn
```

Comment:
Are you sure we need this? I'm almost 100% sure yarn is already available on the Linux image on github. But we can check.

Reply:
no, i left it by accident lemme get rid of it